weatherman.py

```diff
@@ -0,0 +1,286 @@
+import argparse
```

Comment:
Let's refactor this code to make use of classes for calculations and reporting.

Reply:
alright